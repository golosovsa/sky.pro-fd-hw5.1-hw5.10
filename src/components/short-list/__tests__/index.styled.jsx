import ShallowRenderer from 'react-test-renderer/shallow'
import ShortList from '..'

const mockedItems = [
  {
    value: 'value1',
    checked: true
  },
  {
    value: 'value2',
    checked: true
  },
  {
    value: 'value3',
    checked: true
  }
]

describe('<ShortList />', () => {
  describe('Snapshot test', () => {
    it('should render successfully and match snapshot', () => {
      const renderer = new ShallowRenderer()
      const tree = renderer.render(<ShortList items={mockedItems} />)

      expect(tree).toMatchSnapshot()
    })
  })
})
