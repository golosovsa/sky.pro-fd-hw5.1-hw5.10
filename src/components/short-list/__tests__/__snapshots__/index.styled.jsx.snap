// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ShortList /> Snapshot test should render successfully and match snapshot 1`] = `
<styled.div>
  <styled.p
    onClick={[Function]}
  >
    value1
  </styled.p>
  <styled.p
    onClick={[Function]}
  >
    value2
  </styled.p>
  <styled.p
    onClick={[Function]}
  >
    value3
  </styled.p>
</styled.div>
`;
